File: dayTwo/Java/Level 2/TaskFour.java
import java.util.Scanner;

public class TaskFour{
    public static void main(String[] args){

        Scanner input = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int firstNumber = input.nextInt();

        System.out.print("Enter a number again: ");
        int secondNumber = input.nextInt();

        if(firstNumber > secondNumber){
            System.out.println(firstNumber + " is the largest");
        }
        else if(secondNumber > firstNumber){
            System.out.println(secondNumber + " is the largest");
        }
    }
}

File: dayTwo/JavaScript/Level 2/TaskFour.js
const prompt = require("prompt-sync")();

let firstNumber = parseInt(prompt("Enter a number: "));

let secondNumber = parseInt(prompt("Enter a number again: "));

if(firstNumber > secondNumber){
    console.log(firstNumber + " is the largest")
}
else if(secondNumber > firstNumber){
    console.log(secondNumber + " is the largest")
}
